mm2src/mm2_bin_lib/Cargo.toml

```diff
@@ -5,8 +5,8 @@
 
 [package]
 name = "mm2_bin_lib"
-version = "2.5.2-beta"
-authors = ["James Lee", "Artem Pikulin", "Artem Grinblat", "Omar S.", "Onur Ozkan", "Alina Sharon", "Caglar Kaya", "Cipi", "Sergey Boiko", "Samuel Onoja", "Roman Sztergbaum", "Kadan Stadelmann <ca333@komodoplatform.com>", "Dimxy", "Omer Yacine", "DeckerSU", "dragonhound <smk@komodoplatform>", "Devin AI"]
+version = "2.6.0-beta"
+authors = ["James Lee", "Artem Pikulin", "Artem Grinblat", "Omar S.", "Onur Ozkan", "Alina Sharon", "Caglar Kaya", "Cipi", "Sergey Boiko", "Samuel Onoja", "Roman Sztergbaum", "Kadan Stadelmann <ca333@komodoplatform.com>", "Dimxy", "Omer Yacine", "DeckerSU", "dragonhound <smk@komodoplatform>", "gcharang", "Devin AI"]
```

Comment:
@gcharang Do you prefer a different name other than `gcharang` to be added to authors? Since you contributed 2 commits in this next release I thought it should be added :)